README.md

```diff
@@ -21,9 +21,7 @@ Before using the Orka GitHub Runner, ensure that the following prerequisites are
 
 * GitHub App: Having a GitHub App is a prerequisite for using the Orka GitHub Runner. You can find instructions on creating a GitHub App in the [Creating a GitHub app](docs/github-app-setup-steps.md) file.
 * Connectivity to Orka 3.0+ cluster: Ensure that the machine where the Orka Github Runner is started has connectivity to the Orka cluster. Additionally, ensure that the SSH ports are open to enable the runner to establish SSH connections with Orka VMs.
-* The Orka GitHub runner supports GitHub.com hosted environments.
-
-> **NOTE**: Currently, GitHub Enterprise Server is not supported.
+* The Orka GitHub runner has been tested with GitHub.com hosted environments. GitHub Enterprise Server was not available to the development team for testing, but given the runner was heavily inspired by the [ARC project](https://github.com/actions/actions-runner-controller), it is expected to work on GitHub Enterprise Server 3.6+
```

Comment:
I'm not sure we can support GitHub Enterprise at the moment - @ispasov can you confirm?

Reply:
From our previous conversations, I understood that since we are based on the ARC project, we should be compatible, but since we don't have access to an instance for testing, we can't confirm. I'd like to have this posture externally as a group of target users do run the on-prem GHES.

Reply:
@jasoncdavis0 it is not supported.
We are based on the ARC project, but the support for Enterprise Server does not come out of the box. 

We do not have the logic implemented to communicate with hosted servers. For example, we [require](https://github.com/macstadium/orka-github-actions-integration/blob/main/pkg/env/env.go#L169) that the URL people pass starts with github.com.

Reply:
@jasoncdavis0 Could we update the README to note that while this isn't supported, we'd welcome community contributions for it, lightly outline how someone might get started, and people can then open an issue if they'd like?

Reply:
@ispasov  @spikeburton Hi friends! I opened #20 to try to clarify this a bit further. Let me know if that works, or if you'd like any additional changes. Thank you so much! :)